test/database/avert-model.test.ts

```diff
@@ -28,7 +29,7 @@ describe("AvertModel schema validation", () => {
   it("should fail validation for an invalid powerPlantClass", () => {
     const avert = new AvertModel({
       year: 2024,
-      location: EgridLocation.options[0], // Valid location
+      location: AvertLocation.options[0], // Valid location
```

Comment:
Both `EgridLocation.options[0]` and `AvertLocation.options[0]` refer to `"US"` so it's not actually testing the difference between the two enums here. Changing it to a different option would help make sure that we are using the correct enum.